nms/server/grafana/routes.tsx

```diff
@@ -116,7 +116,7 @@ const proxyMiddleware = () => {
       },
       proxyReqPathResolver: req => req.originalUrl.replace(/^\/grafana/, ''),
       userResDecorator: (proxyRes, proxyResData: Buffer, userReq, userRes) => {
-        userRes.set('X-Frame-Options', 'allow');
```

Comment:
Allow is not a valid option https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options